src/subhalo_tracking.cpp

```diff
@@ -864,19 +881,43 @@ class SubhaloMasker_t
     if (subhalo.Nbound <= 1)
       return; // skip orphans
 
+    // Save first tracer so we can add it back if all tracers are masked out
+    auto tracer = subhalo.Particles[0];
+    if (!tracer.IsTracer())
+    {
+      throw runtime_error("No tracer particle found before masking");
```

Comment:
When would this error be thrown if `throw runtime_error("No tracer particle from previous snapshot found in FOF");` isn't thrown?

Reply:
It shouldn't ever be, but decided to leave it in case the newly created centrals are doing something they shouldn't